.github/workflows/check-docs-only.yaml

```
@@ -0,0 +1,40 @@
+# Copyright (c) 2022 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+name: docs-only-check
```

Could you make this a bit more verbose. Maybe something like:

```suggestion
name: Skip the CI if the PR only changes document files.
```